### PR TITLE
Bump kube dependencies to v1.19.0

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,53 +3,53 @@ module github.com/kubernetes-csi/external-resizer
 go 1.15
 
 require (
-	github.com/container-storage-interface/spec v1.2.0
+	github.com/container-storage-interface/spec v1.3.0
 	github.com/imdario/mergo v0.3.9 // indirect
```
Comment (gnufied) [inline go.mod:7]: did you intend to update CSI spec or did the update come via k8s client update?

Comment (humblec) [reply go.mod:7]: @gnufied I deliberately updated it , but forgot to explictly mention it

Comment (gnufied) [reply go.mod:7]: I would rather not update that in this PR.

Comment (gnufied) [reply go.mod:7]: @humblec I see no reason to update CSI spec at last minute within resizer repo. Also have we updated other sidecar's CSI spec version? k8s repo at least is still using 1.3.0.

Comment (humblec) [reply go.mod:7]: almost all sidears are with CSI spec v1.3.0

Comment (humblec) [reply go.mod:7]: @gnufied I can revert the change too.. please let me know the choice

Comment (humblec) [reply go.mod:7]: @gnufied better I reverted..  We will update the SPEC 1.3 accordingly. PTAL

**Pull request merged**